perf/blogbench.py

```diff
@@ -0,0 +1,62 @@
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils import archive
+from avocado.utils.software_manager import SoftwareManager
+from avocado.core import data_dir
+
+
+class Blogbench(Test):
+
+    ''' Blogbench will start the required threads and the test will run
+        according to the args given.A final "score" will then be given as
+        an indication of read and write performance '''
+
+    def setUp(self):
+        sm = SoftwareManager()
+        # Check for basic utilities
+        deps = ['gcc', 'make']
+        for package in deps:
```

Comment:
it can be 

`for package in ['gcc', 'make']:` as eliminate to use extra variable please consider